docs/src/main/paradox/elasticsearch.md

```diff
@@ -30,7 +30,7 @@ Sources, Flows and Sinks provided by this connector need a prepared `org.elastic
 access to Elasticsearch.
```

Comment:
Please add a section about the implemented API versions with a note that it does not need to excatly match the Elasticsearch server version.

Reply:
Added a brief bit into the `Sink and flow configuration` section of the docs. Added links to the corresponding API docs but can go into more detail here if needed.